src/Components/Book.js

```diff
@@ -1,14 +1,47 @@
-/* eslint-disable react/prop-types */
 import PropTypes from 'prop-types';
+import { CircularProgress } from '@material-ui/core';
 
-const Book = ({ book, handleRemoveBook }) => (
-  <tr>
-    <td>{book.id.substring(0, 2)}</td>
-    <td>{book.title}</td>
-    <td>{book.category}</td>
-    <td><button type="button" onClick={() => handleRemoveBook(book)}>Remove book </button></td>
-  </tr>
-);
+const Book = ({ book, handleRemoveBook }) => {
+  const progressBar = Math.floor(Math.random() * 100);
+  const progressPage = Math.floor(Math.random() * 500);
+
+  return (
+    <div className="book-item">
+      <div className="book-info">
+        <h4>{book.category}</h4>
+        <h3>{book.title}</h3>
+        <h5>Author</h5>
```

Comment:
Hello TSE. I have to triple check on this. However, the problem relies on that you are using a small screen size. This project does not require any responsiveness, therefor the page is set to be on full desktop display in order to not overlaps, otherwise it will require a responsive design which will requite a complete different styling. I have fixed some of this, but please bear in mind this is due to the screen size you have. 

![image](https://user-images.githubusercontent.com/40170054/127610835-f81dd39d-77a5-4a01-aa46-242695b33c01.png)